spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/StreamEnvironmentPostProcessor.java

```diff
@@ -44,8 +43,7 @@ public void postProcessEnvironment(ConfigurableEnvironment environment,
 			SpringApplication application) {
 		Map<String, Object> map = new HashMap<String, Object>();
 		map.put("spring.cloud.stream.bindings." + SleuthSink.INPUT + ".content-type",
-				environment.getProperty("spring.sleuth.stream.content-type",
-						"application/x-java-object;type=" + Spans.class.getName()));
+				"");
```

Comment:
Will do. To be frank, I am not very happy with this approach. I'd rather remove this postprocessor completely, but it is necessary because the other `StreamEnvironmentPostProcessor` that we use for producers is also on the classpath and sets the `content-type` of the input channel to `application/json`. Maybe we could consider using separate channel names for input and output and set configurations separately.